server/api/availability_handler.py

```diff
@@ -0,0 +1,113 @@
+import json, re
+from flask import jsonify, request, Blueprint, session
+from datetime import datetime
+from api.login_decorator import login_required
+from pytz import all_timezones
+
+# Models
+from models.shared import db
+from models.user import User
+from models.availability import Availability
+
+
+availability_handler = Blueprint('availability_handler', __name__)
+@availability_handler.route('/availabilities', methods=["GET"])
+@login_required
+def availability_get():
+    sess = dict(session)
+    user = User.query.filter_by(email=sess["profile"]["email"]).first()
+
+
+    if user is None:
+        return jsonify({
+            "status": "error",
+            "message": "Please complete setting up your profile first."
+        }), 400
+    # We need to output the following:
+    # - name, description, duration
+    # Should we only display upcoming appointments or show everything?
+    # should we use query parameters to limit the output and or view based on date?
+    output = []
+    # Get just the meeting ids, no need for other fields
+    availabilities = Availability.query.filter_by(user_id=user.id)
+    for availability in availabilities:
+        output.append({
+            "id": availability.id,
+            "day": availability.day,
```

Comment:
Let's discuss this in standup today - I think we may be using diff terminologies in diff places

Reply:
Availability as in the timeframes the user is available or the slots they have set where they are "available" for meetings. Availability model has the layout for that.

Reply:
Discussed this in meeting - resolved. this is fine to me :)